.github/workflows/sendReassurePerfData .yml

```diff
@@ -0,0 +1,43 @@
+name: Send Reassure Performance Tests to Graphite
+
+on:
+  push:
+    branches: [main]
+    paths-ignore: [docs/**, contributingGuides/**, jest/**, workflow_tests/**]
+
+jobs:
+  perf-tests:
+    if: ${{ github.actor != 'OSBotify' }}
+    runs-on: ubuntu-latest-reassure-tests
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup NodeJS
+        uses: ./.github/actions/composite/setupNode
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run performance testing script
+        shell: bash
+        run: |
+          set -e
+          npx reassure --baseline
+
+      - name: Get merged pull request
+        id: getMergedPullRequest
+        uses: actions-ecosystem/action-get-merged-pull-request@59afe90821bb0b555082ce8ff1e36b03f91553d9
```

Comment:
last released 4 years ago, are there any alternatives? It just may be stable, but I'm still curious.

Reply:
It is exactly the one we use in the `preDeploy` workflow:
https://github.com/Expensify/App/blob/e4eead376187181ed5e7d22fad866f05a6dd09aa/.github/workflows/preDeploy.yml#L47

So if we use and it works fine, don't think it is necessary to change it